.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
+
+on:
+  [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Touch the env file 
+      run: touch deployment_files/.env
```

Comment:
You don't need this step. The .env file is useful for local dev, but your pipeline provides env vars to the container via `-e ...` instead.